android/feature-now-playing/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id(Plugins.androidLibrary)
+    id(Plugins.hilt)
+    kotlin(Plugins.android)
+    kotlin(Plugins.kapt)
+}
+
+android {
+    compileSdk = Versions.compileSdk
+
+    defaultConfig {
+        minSdk = Versions.minSdk
+        targetSdk = Versions.targetSdk
+        resourcePrefix = "now_playing"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    buildFeatures {
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = Versions.composeCompiler
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+}
+
+dependencies {
+    kapt(Hilt.hiltCompiler)
+    kapt(Hilt.hiltAndroidCompiler)
+    kapt(Airbnb.showkaseProcessor)
+
+    implementation(project(Modules.coreTheme))
+    implementation(project(Modules.shared))
+
+    implementation(Airbnb.showkase)
+    implementation(Google.androidxCore)
+    implementation(Google.appCompat)
+    implementation(Google.composeMaterial)
+    implementation(Google.composeUi)
+    implementation(Google.composeUiToolingPreview)
+    implementation(Google.constraintLayout)
```

Comment:
Are we using the constraintLayout somewhere?

Reply:
This may be an artifact of the pre-jetpack days and copy pasta here.  I want to go through and create some gradle scripts that make it so we don't have to copy paste every dependency everywhere, so I'll clean this up later.